### DISCUSSION TITLE
ResponsiveContainer does not render in react-bootstrap components

### DISCUSSION THREAD

**Discussion by peymanahmadi** [recharts/recharts#2763] [Q&A] [answered]
In my case, I have problems in working with ResponsiveContainer. For example, while using react-bootstrap ResponsiveContainer does not render any chart, at least I couldn't. Has anybody had a problem like this?

**Comment by DennyPlank** [comment]
Did you ever solve this? It's been failing to render for me as well for a while now. Some people have said you need to set ResponsiveContainer heights and widths but that didn't fix my issue.

**Reply by didje251** [reply]
Few months later, and after looking into the Issues for any fix for this and I'm still having this problem 😢  Anytime I want to use a ResponsiveContainer I get an error. I use the latest Recharts version with the latest React version...

Uncaught TypeError: Cannot read properties of null (reading 'useState')

**Reply by mengs98** [reply]
Hi, I recently had the same problem while making a full-stack app, but it is solved now! The problem with my project was, I installed the recharts package on the server, not on the client's package.json file. Maybe you can check yours.

**Comment by ckifer** [comment, marked as answer]
Its tough to take a look at something like this without a reproduction since recharts doesn't use `react-bootstrap`. 

Looks like in @mengswe 's case it was due to server side rendering (https://github.com/recharts/recharts/discussions/2763#discussioncomment-4899802). Please make sure that is not the case. 

If someone wants to create a reproduction I will look into it
